synapse/handlers/room.py

```diff
@@ -1079,9 +1037,6 @@ async def _send_events_for_new_room(
         # created (but not persisted to the db) to determine state for future created events
         # (as this info can't be pulled from the db)
         state_map: MutableStateMap[str] = {}
-        # current_state_group of last event created. Used for computing event context of
-        # events to be batched
-        current_state_group: Optional[int] = None
```

Comment:
This was a bit of drive-by cleanup, I noticed that we are no longer using this as it's reassigned below. I can put this in a separate PR if that's distasteful.